### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by livelxw** [monix/monix-kafka#81]
Are there any plans for a new release? I think the manual commit might be useful

**Comment by poslegm** [comment]
There is snapshot release with manual commits:
```
"io.monix" %% "monix-kafka-1x" % "1.0.0-RC2-SNAPSHOT-48d9da7-SNAPSHOT"
```

**Comment by Avasil** [comment]
I will make a normal release today. Probably `1.0.0-RC2.1`, `Monix 3.0.0-RC3` might take a bit more time

**Comment by Avasil** [comment]
@livelxw I've released `1.0.0-RC3`
